src/core.c/Backtrace.rakumod

```diff
@@ -171,9 +174,11 @@ my class Backtrace {
             next if $file.ends-with('BOOTSTRAP.nqp')
                  || $file.ends-with('QRegex.nqp')
                  || $file.ends-with('Perl6/Ops.nqp');
-            if $file.ends-with('NQPHLL.nqp')
+
+            my $name := nqp::p6box_s(nqp::getcodename($do));
+
+            if ($file.starts-with('NQP::') && $file.ends-with('Compiler.nqp') && $name eq 'eval')
```

Comment:
Should this have always been been `$file.starts-with('NQP/')`? It seems like a file containing `::` might not be intentional

Reply:
I guess there are other examples like `SETTING::`, so I suppose it must be correct

Reply:
You're correct. `SETTING::` and `NQP::` are special prefixed introduced by composing script via `#line` directive. Very helpful when categorizing files.